tests/Mocks/Tenants/Factories/Legacy/MultiFactory.php

```diff
@@ -41,6 +42,7 @@
     NullHttpTenant::class,
     NullMixedTenant::class,
     NullQueueTenant::class,
+    PgsqlTenant::class,
```

Comment:
Weird indenting?

Reply:
I rebased from my old branch onto 1.x and didn't check... #whoops

Reply:
Actually, looking again, I don't see anything weird about it.  My phone did though.  Guessing it's just github being weird

Reply:
Jup had the same, looked at PC and everything was fine